websites/docs/src/content/docs/en/getting_started/installation.mdx

```diff
@@ -41,6 +54,69 @@ Before you can install and use Ballerine, you will need the following:
 
+### Installation steps on a remote virtual machine
+
+We recommend installation using ansible
+
+
+1. **Install ansible**: We recommend installing ansible with apt:
+<CodeBlock lang={`shell`}  code={`apt install -y ansible`}/>
+
+2. **Clone the project**: Use Git to clone the Ballerine repository to your local machine:
+<CodeBlock lang={`shell`}  code={`git clone https://github.com/ballerine-io/ballerine.git && cd ballerine`}/>
+
+3. **Switch to the dev branch**: After cloning, switch to the development branch:
+<CodeBlock lang={`shell`}  code={`git checkout dev`}/>
+
+4. **Navigate to ballerine_playbook directory**:
+<CodeBlock lang={`shell`}  code={`cd deploy/ansible/ballerine_playbook`}/>
+
+5. **Create inventory file**:
+<CodeBlock lang={`shell`}  code={`touch inventory.txt`}/>
+
+Now, with your editor, open the file and add the hostname or FQDN of the server(s) you want to deploy Ballerine to with the following pattern:
+
+6. **Add entries into the inventory file**:
+<CodeBlock lang={`shell`}  code={`all ansible_host={{ SERVER_HOST }} ansible_port={{ SERVER_PORT }} ansible_user={{ SERVER_USER }}`}/>
+
+If you are using SSH keypairs for authenticating your SSH connections to your server. You can tell Ansible your ssh private key file in the `inventory` file
+using `ansible_ssh_private_key_file`
+
+<CodeBlock lang={`shell`}  code={`all ansible_host={{ SERVER_HOST }} ansible_port={{ SERVER_PORT }} ansible_user={{ SERVER_USER }} ansible_ssh_private_key_file={{ SSH_PRIVATE_KEY_FILE }}`}/>
+
+After you completed the above step then we're pretty much done with the inventory
+
+7. **Setup your configuration vars for Ballerine**:
+
+The next step is to setup necessary configuration for your app to run such as environment variable, domain name, etc.
+
+First you need to open `deploy/ansible/ballerine_playbook/roles/setup-ballerine/defaults/main.yml` file with your editor.
+There are some variables that will need input from you to get the application start correctly
+
+- `install_dir`: The absolute path of your app's installation folder on the server (required). Default: `/home/ubuntu/ballerine`
+- `vite_api_url`: Incase you want to deploy ballerine on a remote server and run it on HTTPS
+- `backoffice_url`: URL you wish to deploy Case-Management on
+- `kyb_url`: URL you wish to deploy KYB on
+- `workflow_dashboard_url`: URL you wish to deploy Workflows-Dashboard on
+- `workflow_svc_url`: URL you wish to deploy Workflows-Service on
+
+Once you complete setup config vars for your app then we are ready to deploy our app on your server.
+
+8. **Run the Ansible playbook**:
+
+After complete the above step. Now the only remain step we need to do is run the ansible playbook.
+You can run the ansible playbook with the following command
+
+<CodeBlock lang={`shell`}  code={`cd ballerine/deploy/ansible/ballerine_playbook; ansible-playbook -i inventory.txt ballerine-playbook.yml`}/>
```

Comment:
Most of our non-React files in the monorepo are kebab-case, don't think we should be out of the convention unless its some Ansible specific convention.
```suggestion
<CodeBlock lang={`shell`}  code={`cd ballerine/deploy/ansible/ballerine-playbook; ansible-playbook -i inventory.txt ballerine-playbook.yml`}/>
```

Reply:
This is more of ansible convention. So, went with it. Also change in this means change in actual code now